@vates/nbd-client/multi.mjs

```diff
@@ -34,9 +34,8 @@ export default class MultiNbdClient {
   async connect() {
     const candidates = [...this.#settings]
 
-    const promises = []
     const baseOptions = this.#options
-    const  _connect = async () => {
+    const _connect = async () => {
       if (candidates.length === 0) {
         return
       }
```

Comment:
Is the randomization (lines 42-43) still necessary now that we connect sequentially?

I assumed this was done to discern multiple simultaneous clients, but I could be wrong.

Reply:
Yes because each connection will be on a random host, hopefully equally distributed